docs/content/guide/api.md

```diff
@@ -194,6 +194,14 @@ Runs a command on the node. Returns an instance of `bundlewrap.operations.RunRes
 -   `may_fail` If `False`, `bundlewrap.exceptions.RemoteException` will be raised if the command does not return 0.
 
 `bundlewrap.exceptions.TransportException` will be raised if there was a transport error while running the command, e.g. the SSH process died unexpectedly.
+=======
+
+<br>
+
+**`.run_ipmitool(command)`**
+
+Runs a command on an ipmi interface. Returns an instance of
```

Comment:
This should explain what a “command” is. That should probably include that the `command` is supposed to be a single string, split at whitespace into tokens.